### PR TITLE
Add support for scoped packages

### PR DESCRIPTION
**Pull request by BBlackwo**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,26 +1,34 @@
...
-    "npm": "^4.2.0",
-    "rimraf": "^2.5.4"
+    "async": "2.1.4",
+    "compressing": "1.1.0",
+    "graceful-fs": "4.1.11",
+    "lodash.curry": "4.1.1",
+    "mute": "2.0.6",
+    "mv": "2.1.1",
+    "npm": "4.2.0",
+    "rimraf": "2.5.4"
```
Comment (BBlackwo) [inline package.json]: I dropped the `^` as there is no `package.lock.json` file. So this was an easy way to lock them down.

Comment (BBlackwo) [reply package.json]: I can add a `package.lock.json` if you'd like me to

**Comment by necccc** [comment]
this is great, thank you!

**Comment by necccc** [comment]
released

**Pull request merged**